tests/subsys/canbus/isotp/conformance/src/main.c

```diff
@@ -813,7 +813,7 @@ ZTEST(isotp_conformance, test_stmin)
 
 	start_time = k_uptime_get_32();
 	ret = k_msgq_get(&frame_msgq, &raw_frame,
-			 K_MSEC(STMIN_VAL_1 + STMIN_UPPER_TOLERANCE));
+			 K_TICKS(k_ms_to_cyc_ceil32(STMIN_VAL_1 + STMIN_UPPER_TOLERANCE)));
```

Comment:
Not entirely sure if I got this right, but it looks like `k_ms_to_cyc_ceil32` returns hardware cycles, which are configured as `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=1000000` for `native_posix`. However, `K_TICKS` expects system clock ticks, which are based on [`SYS_CLOCK_TICKS_PER_SEC`](https://github.com/zephyrproject-rtos/zephyr/blob/63850804f167353df27b6b33d05f8f3ab642aae4/kernel/Kconfig#L614) and default to `100` for `native_posix`. Thus, `K_TICKS` will get a comparably huge number and not time out anymore.

Using `k_ms_to_ticks_ceil32` should lead to the same result as `K_MSEC` as far as I can see, so it would not fix the problem.

Maybe increasing `CONFIG_SYS_CLOCK_TICKS_PER_SEC` as done in [samples/bluetooth/unicast_audio_client](https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/unicast_audio_client/boards/native_posix.conf) would be more appropriate?

Reply:
Thanks. I have changed it to `CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000` as per our conversations in Discord. I have put this PR back in draft state until I have had a chance to run the branch on our CAN board test farm.